test/e2e/load_test.go

```diff
@@ -217,4 +217,20 @@ var _ = Describe("Podman load", func() {
 		Expect(result.LineInOutputContains("docker")).To(Not(BeTrue()))
 		Expect(result.LineInOutputContains("localhost")).To(BeTrue())
 	})
+
+	It("podman load xz compressed image", func() {
+		outfile := filepath.Join(podmanTest.TempDir, "alp.tar.xz")
+
+		save := podmanTest.Podman([]string{"save", ALPINE, "|", "xz", "-zc", "-", ">", outfile})
```

Comment:
(Looking at only the `xz` failures, not things like “[Fail] Podman logs [It] podman logs for container ”.)

AFAICS `PodmanTest`.PodmanAsUser`, uses `exec.Command`:
> Unlike the "system" library call from C and other languages, the os/exec package intentionally does not invoke the system shell and does not expand any glob patterns or handle other expansions, pipelines, or redirections typically done by shells.

i.e. the `|`… are treated as additional arguments to `podman save`, and are passed to `libpodImage.GetAdditionalTags`.  Passing `|` to this function would explain the
> error parsing additional tags: invalid reference format

error.

That could explain the error in CI — and I think the test suite would fail when run locally as well.

Running ”the same line” in shell does not actually run the same command when the `|` and `>` are not quoted.

Reply:
Thank you! I changed the test and got rid of the pipe.